msteams-platform/sbs-create-dashboard-widget-graph-api-call-in-Teams-toolkit.yml

```diff
@@ -0,0 +1,280 @@
+         return (
+         <>
+         <ListWidget />
+         <ChartWidget />
+         <SampleWidget />
+         </>);
+         } 
+        ```
+
+        :::image type="content" source="~/assets/images/sbs-create-a-new-dashboard/create-sample-dashboard.png" alt-text="Screenshot shows the existing sampleDashboard file in Visual Studio Code."::: 
+
+      If you want the widget in a column, you can use the `oneColumn()` method to define the column layout.
+      
+      Following is the sample of `oneColumn()` layout:
+
+        ```
+  
+        protected dashboardLayout(): void | JSX.Element {
+        return (
+        <>
+        <ListWidget />
+        <div style={oneColumn()}>
+                  <SampleWidget />
+                  <ChartWidget />
+                </div>
+              </>
+            );
+          }
+        ```
+
+      :::image type="content" source="~/assets/images/sbs-create-a-new-dashboard/one-column.png" alt-text="Screenshot shows the addition of one column method in dashboard layout.":::
+
+      After adding all the relevant details for creating a widget, Save the respective files.
+
+      You can view the layout successfully changed on the app and view the calendar widget on the dashboard.
+
+      :::image type="content" source="~/assets/images/sbs-create-a-new-dashboard/widget-added-to-dashboard.png" alt-text="Screenshot shows the addition of widget to the dashboard with the sample widget highlighted in red. ":::
+
+      You can change the look and style of the widget. For more information, see [customize the widget](tabs/how-to/dashboard-teams-toolkit.md#customize-the-widget).
+
+- title: Add a new Graph API call
+  durationInMinutes: 5
+  content: |
+
+      1. Add **single-sign-On** related files by using Teams toolkit.
+
+      1. In Visual Studio Code, select **Teams** in the left pane.
+      
+      1. Under **DEVELOPMENT**, select **Add features**.
+
+          :::image type="content" source="~/assets/images/sbs-create-a-new-dashboard/add-features.png" alt-text="Screenshot shows the selection of added features option.":::
+
+      1. Select **Single Sign-On**.
+
+          :::image type="content" source="~/assets/images/sbs-create-a-new-dashboard/single-sign-on.png" alt-text="Screenshot shows the selection of single sign-on option.":::
+
+      1. Move `auth-start.html` file and `auth-end.html` file in the auth> tab> public folder to the tabs> public> folder. These two HTML files are used for auth redirects.
+
+      1. Move **SSO** folder under **auth > tab** folder to tabs> src> sso folder.
+
+      You've added the single-sign-on files to your project, you can call Graph APIs. Refer to [Graph API call](tabs/how-to/dashboard-teams-toolkit.md#graph-api-call).
+
+      1. Select **Join** to join the meeting.
+
+         :::image type="content" source="assets/images/sbs-create-a-new-dashboard/join.png" alt-text="Screenshot shows the Join button highlighted in red.":::
```

Comment:
The `join.png` image seems not very clear, I have a clearer picture, please feel free to replace it.

![image](https://user-images.githubusercontent.com/107838226/215303404-09a21c93-6628-43cb-8bb6-ff3b2bc4bbd8.png)